### PR TITLE
vscode-extensions.zhwu95.riscv: init at 0.0.8

### PR DESCRIPTION
**Pull request by CardboardTurkey**
###### Description of changes

Provides basic RISC-V colorization and snippets support: https://github.com/zhuanhao-wu/vscode-riscv-support

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -2361,6 +2361,12 @@
     github = "CaptainJawZ";
     githubId = 43111068;
   };
+  CardboardTurkey = {
+    name = "Kiran";
```
Comment (msm-code) [inline maintainers/maintainer-list.nix:2365]:
```suggestion
    name = "Kiran Ostrolenk";
```
I suppose (unless that's intentional to be anonymous, but the last name is in the email anyway).

**Comment by msm-code** [comment]
Result of `nixpkgs-review pr 213607` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>vscode-extensions.zhwu95.riscv</li>
  </ul>
</details>

File: pkgs/applications/editors/vscode/extensions/default.nix [hunk 2]
```diff
@@ -3089,6 +3089,22 @@ let
...
+          publisher = "zhwu95";
+          version = "0.0.8";
+          sha256 = "sha256-PXaHSEXoN0ZboHIoDg37tZ+Gv6xFXP4wGBS3YS/53TY=";
+        };
+        meta = with lib; {
+          description = "Basic RISC-V colorization and snippets support.";
+          downloadPage = "https://marketplace.visualstudio.com/items?itemName=zhwu95.riscv";
+          homepage = "https://github.com/zhuanhao-wu/vscode-riscv-support";
+          license = licenses.mit;
+          maintainers = with maintainers; [ CardboardTurkey ];
```
Comment (msm-code) [inline pkgs/applications/editors/vscode/extensions/default.nix:3104]:
```suggestion
          maintainers = maintainers.CardboardTurkey;
```

**Comment by CardboardTurkey** [comment]
@jonringer any chance of a review pretty please? 🤞

**Comment by CardboardTurkey** [comment]
@NickCao I noticed you approved some `vscode-extensions` PRs recently. Any chance you can look at mine? My assigned reviewer isn't responding

**Comment by NickCao** [comment]
Please split the changes into two commits.
1. maintainers: add CardboardTurkey
2. vscode-extensions.hwu95.riscv: init at 0.0.8 
Otherwise this looks good.

**Comment by NickCao** [comment]
@ofborg build vscode-extensions.hwu95.riscv

**Comment by NickCao** [comment]
And there is a typo in the commit message/pr title? hwu95 -> zhwu95

@ofborg build vscode-extensions.zhwu95.riscv

**Comment by CardboardTurkey** [comment]
woops 😬 fixed now

**Review by NickCao — APPROVED** [review]

**Comment by CardboardTurkey** [comment]
Thanks for your help @NickCao @msm-code!

**Pull request merged**